packages/block-library/src/html/preview.js

```diff
@@ -32,7 +33,12 @@ export default function HTMLEditPreview( { content, isSelected } ) {
 
 	return (
 		<>
-			<SandBox html={ content } styles={ styles } />
+			<SandBox
+				html={ content }
+				styles={ styles }
+				title={ __( 'Custom HTML Preview' ) }
+				tabIndex={ -1 }
```

Comment:
Added this because focus should not enter iFrame unless user wants it to. There is no way to escape the iFrame currently since writing flow is unaware of it.

Reply:
Is there an easy fix for this? I guess not since the secondary iFrame is a different browser context.